translations/src/loadTranslations.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { merge } from 'lodash'
+import defaultTranslations from '../translations.json'
+import transformTranslations from './transformTranslations'
+import type { TranslationsType, TransformedTranslationsType } from './types'
+
+const loadTranslations = (translationsOverride: TranslationsType): TransformedTranslationsType => {
```

Comment:
Isn't this duplicated code?

Reply:
Yes this will be replaced in an other PR